catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/transformer/TransformerManager.java

```diff
@@ -99,6 +120,31 @@ public TransformerManager(List<ServiceReference> serviceReferences) {
         return null;
     }
 
+    /**
+     * Returns a list of property maps for transformers that match the given property and value
+     *
+     * @param property The transformer property name
+     * @param value    The value of the transformer property to match
+     * @return List of property maps for the matching transformers
+     */
+    public List<Map<String, Object>> getRelatedTransformerProperties(String property,
+            String value) {
+        List<Map<String, Object>> properties = new ArrayList<>();
+        for (ServiceReference serviceRef : serviceRefs) {
+            Object propertyObject = serviceRef.getProperty(property);
+            if (propertyObject != null && propertyObject instanceof String) {
+                if (value.equals(propertyObject)) {
```

Comment:
In fact, with this test here, you can skip the whole `instanceof` condition and reduce cyclomatic complexity on this method. This one test alone is dispositive.